tools/github-event-processor/Azure.Sdk.Tools.GitHubEventProcessor/Constants/ActionConstants.cs

```diff
@@ -0,0 +1,37 @@
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Azure.Sdk.Tools.GitHubEventProcessor.Constants
+{
+    /// <summary>
+    /// Action constants for github actions. These match the activity types for events defined in the
+    /// github docs https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows.
+    /// Some activities belong to multiple events, for example issues and pull_requests both have an
+    /// opened event.
+    /// </summary>
+    public class ActionConstants
+    {
+        public static readonly string Assigned = "assigned";
```

Comment:
Originally, we weren't sure if piece of what was being coded were going to be common, like having all the actions stuff in a separate library. In that case you'd actually want static readonly because they'd be treated like a ref in a second assembly. The reason why the EventConstants had to be constants is because the switch statement requires constants. I can make them all constants. I'll update this with the next commit.